src/components/pollutedLocations/PollutedLocationCard.tsx

```diff
@@ -18,7 +19,9 @@ const PollutedLocationCard: React.FC<PollutedLocation> = ({
 
   return (
     <CardLayout
-      title={<h2 className="text-lg">{"Title is missing"}</h2>}
+      title={
+        <h2 className="text-lg">{location?.title || "Title is missing"}</h2>
```

Comment:
We result "" and not `null` or `undefined' if the title is missing. Will that work?